config/default.js

```diff
@@ -55,5 +55,6 @@ module.exports = {
     domain: 'mozilla-iot.org',
     pagekite_cmd: './pagekite.py',
     port: 443
-  }
+  },
+  gateway: 'https://localhost:4443'
```

Comment:
This string is a configuration for `node-fetch` for the rules engine's server code to talk to the gateway. This means it doesn't break when the server is tunneling. Thank you for pointing this out because there's a bug in the client-side javascript of the rules engine where this same string occurs when it is subject to the tunnel